### PR TITLE
docs(connector-alipay-web): add Alipay web readme

### PR DESCRIPTION
**Pull request by darcyYe**
## Summary

Add Alipay Web README.md (en-US).

## Testing

UTs.

### CODE REVIEW

File: packages/connector-alipay/README.md [hunk 1]
```diff
@@ -1,2 +1,41 @@
...
+
+## Settings
+
+|    Name    |     Type    |   Enum values   |
+|:----------:|:-----------:|:---------------:|
+|    appId   |    string   |       N/A       |
+| privateKey |    string   |       N/A       |
+|  signType  | enum string | 'RSA' \| 'RSA2' |
+
+## Reference
```
Comment (IceHe) [inline packages/connector-alipay/README.md:38]:
```suggestion
## References
```

File: packages/connector-alipay-native/README.md [hunk 2]
```diff
@@ -27,6 +27,14 @@ Alipay Native connector works closely with Logto SDK on mobile platforms. It tak
...
 
+## Settings
+
+|    Name    |     Type    |   Enum values   |
+|:----------:|:-----------:|:---------------:|
+|    appId   |    string   |       N/A       |
+| privateKey |    string   |       N/A       |
+|  signType  | enum string | 'RSA' \| 'RSA2' |
+
 ## Reference
```
Comment (IceHe) [inline packages/connector-alipay-native/README.md:38]: align with Google and Facebook docs

```suggestion
## References
```

**Review by IceHe — APPROVED** [review]
Overall LGTM

File: packages/connector-alipay/README.md [hunk 3]
```diff
@@ -1,2 +1,41 @@
...
+
+Alipay Web connector is designed for desktop Web applications. It takes advantage of Alipay's OAuth 2.0 authentication workflow and enables Alipay users to sign in to other Apps using public Alipay user profiles without going through a troublesome register process.
+
+## Register Alipay developer account
+
+[Register an Alipay developer account](https://certifyweb.alipay.com/certify/reg/guide#/) if you don't have one.
+
+## Create and configure Alipay app
+
+1. Sign in to the Alipay console with the account you have just registered.
```
Comment (wangsijie) [inline packages/connector-alipay/README.md:11]: Better add a console link, it may be hard to find.

**Review by wangsijie — APPROVED** [review]

File: packages/connector-alipay/README.md (hunk 3 above)
Comment (darcyYe) [reply packages/connector-alipay/README.md]: Good, fixed.

File: packages/connector-alipay-native/README.md (hunk 2 above)
Comment (darcyYe) [reply packages/connector-alipay-native/README.md]: Updated.

File: packages/connector-alipay/README.md (hunk 1 above)
Comment (darcyYe) [reply packages/connector-alipay/README.md]: Updated.

**Pull request merged**